tools/shared/src/main/scala/org/scalajs/core/tools/linker/analyzer/Analyzer.scala

```diff
@@ -85,6 +85,8 @@ private final class Analyzer(config: CommonPhaseConfig,
         case CyclicDependencyException(chain, from) =>
           _errors += CycleInInheritanceChain(chain, from)
       }
+
+      checkConflictingExports()
```

Comment:
Consider putting this inside the `try/catch` for cyclic dependency errors. When a cyclic dependency error happens, things become pretty fragile and should terminate ASAP. The particular case of `checkConflicitingExports()` is safe, but I would prefer if we kept the codebase in a state where it's clear that nothing should happen after we've detected a cyclic dependency.